test/framework/operator.go

```diff
@@ -69,6 +72,17 @@ var (
 			crdName:             "keycloaks.keycloak.org",
 		},
 	}
+
+	// CommunityCatalog operator catalog for community
+	CommunityCatalog = operatorCatalog{
+		source:    "community-operators",
+		namespace: "openshift-marketplace",
+	}
+	// OperatorHubCatalog operator catalog of Operator Hub
+	OperatorHubCatalog = operatorCatalog{
+		source:    "operatorhubio-catalog",
+		namespace: "olm",
+	}
```

Comment:
SMELL: I don't think this is related to kubernetes or openshift.... However, minikube installs the operatorhubio catalog in the olm namespace. I need to see if we can install the openshift-marketplace in Kubernetes.